neuromation/api/parser.py

```diff
@@ -64,3 +102,53 @@ def _remote_to_local_image(self, image: RemoteImage) -> LocalImage:
             self._config.username, self._config.cluster_name, self._config.registry_url
         )
         return parser.convert_to_local_image(image)
+
+    def env(
+        self, env: Sequence[str], env_file: Sequence[str] = ()
+    ) -> Tuple[Dict[str, str], Dict[str, URL]]:
+        env_dict = self._build_env(env, env_file)
+        secret_env_dict = self._extract_secret_env(env_dict)
+        return env_dict, secret_env_dict
+
+    def _build_env(
+        self, env: Sequence[str], env_file: Sequence[str] = ()
+    ) -> Dict[str, str]:
+        lines: List[str] = []
+        for filename in env_file:
+            lines.extend(_read_lines(filename))
+        lines.extend(env)
+
+        env_dict = {}
+        for line in lines:
+            splitted = line.split("=", 1)
+            name = splitted[0]
+            if len(splitted) == 1:
+                val = os.environ.get(splitted[0], "")
+            else:
+                val = splitted[1]
+            env_dict[name] = val
+        return env_dict
+
+    def _extract_secret_env(self, env_dict: Dict[str, str]) -> Dict[str, URL]:
+        secret_env_dict = {}
+        for name, val in env_dict.copy().items():
+            if val.startswith("secret:"):
+                secret_env_dict[name] = self._parse_secret_resource(val)
+                del env_dict[name]
+        return secret_env_dict
+
+    def volumes(self, volume: Sequence[str]) -> Tuple[Set[Volume], Set[SecretFile]]:
```

Comment:
Please replace `Set[...]` with `Sequence[...]` in the signature and return a pair of lists actually.
The reason is: `api.Container` has sequence types for `volumes` and `secret_files`, let's keep the consistency.

Reply:
`List[...]` is even more appropriate data type, as you use `Dict` for envs

Reply:
Done, but I had to add `volumes = list(volumes)` in cli code in order to make the var mutable.